app/javascript/tests/mocks/mockCloseContact.js

```diff
@@ -0,0 +1,43 @@
+const mockCloseContact1 = {
+  id: 0,
```

Comment:
This is technically an invalid id (they start at 1) - would probably be good to change.

Reply:
Corrected.

Reply:
ID can never be null either. It's not a big deal but why not just make it 1 and the other one 2?

Reply:
Agreed if the CloseContact exists, but if they dont, this will start off as `null`. 
`{this.props.close_contact.id ? 'Update' : 'Create'}`